roles/neutron-common/meta/main.yml

```diff
@@ -27,3 +27,10 @@ dependencies:
     dependent_packages: "{{ neutron.osp_sys_packages }}"
     rootwrap: True
     when: openstack_install_method == 'distro'
+  - role: openstack-distro
+    project_name: neutron-lbaas
+    dependent_packages: "{{ neutron.osp_sys_packages }}"
+    rootwrap: True
```

Comment:
I don't think we actually need rootwrap on openstack-distro.  Looks like we us it in a few places though so we can probably leave here and come back and cleanup if needs be.